Neos.Neos/Configuration/Settings.yaml

```diff
@@ -422,7 +422,7 @@ Neos:
           Json: Neos\Eel\Helper\JsonHelper
           I18n: Neos\Flow\I18n\EelHelper\TranslationHelper
           q: Neos\Eel\FlowQuery\FlowQuery::q
-          Neos.Node: Neos\Neos\Fusion\Helper\NodeHelper
+          Neos.Node: Neos\Neos\Fusion\Helper\NodeLabelHelper
```

Comment:
This doesn't look right to me

Reply:
jup its to say a litte dirty at least. The `NodeHelper` contained in Neos 8.3 just 3 methods but in Neos 9 now a whole lot more (and also the `label` method which would lead to a recursion)
Alternatively we could introduce a new namespace like `Neos.NodeLabel.labelForNode` or `labelForNode(node)` as function, or `NodeLabel.labelForNode` while probably still keeping the proposed `Neos.Node: Neos\Neos\Fusion\Helper\NodeLabelHelper` as legacy layer.

But i thought its not really worth the effort. Thats why the documentation in `NodeLabelHelper` also explains its not the same as the other. Whats your idea.

Reply:
No this is confusing as hell, please lets not do this

Reply:
Jup agreed. Hmm especially here:

https://github.com/neos/neos-development-collection/blob/6bbc133504ca71772d0f079bdbdf7b58348a23bc/Neos.Neos/Classes/Domain/NodeLabel/ExpressionBasedNodeLabelGenerator.php#L44

We actually need the node type in the yaml expression ...

so i guess there are two ways from here:

1.

introduce a new `NodeLabelHelper` an register it under `Neos.NodeLabel` or something
keep the `Neos.Node` helper registered in yaml but deprecate its method `labelForNode`

2.

Keep everything in `NodeHelper` and just rename `labelForNode` 

3

Keep `labelForNode` and `label` side by side which is a little confusing but maybe okay?

Reply:
we decided on 3